delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/OrganizationsRoutes.scala

```diff
@@ -49,9 +56,30 @@ final class OrganizationsRoutes(identities: Identities, organizations: Organizat
       OrganizationSearchParams(deprecated, rev, createdBy, updatedBy)
     }
 
+  private def authorizeForOrgUUID(uuid: UUID, permission: Permission)(implicit
```

Comment:
As it also gets the org, fetchByUuid or something similar ?

Reply:
Since it also does the authorization then it should probably be `fetchAndAuthorizeForOrgByUUID` which is getting quite long, but could work.

Reply:
Authorization is part of the job of routes so it is somewhat implicit especially if you pass along a permission as parameter.
And we are in Organizations routes, so we could drop the Org